Flaky test: DockerTrustSuite.TestBuildContextDirIsSymlink

Found in https://github.com/docker/docker/pull/30041 (EDIT: also in https://github.com/docker/docker/pull/29965)

https://jenkins.dockerproject.org/job/Docker-PRs-experimental/29414/consoleFull

```
01:14:09.822 --- FAIL: Test (3427.89s)
01:14:09.825 panic: DockerTrustSuite.TestBuildContextDirIsSymlink test timed out after 5m0s [recovered]
01:14:09.826 	panic: DockerTrustSuite.TestBuildContextDirIsSymlink test timed out after 5m0s
01:14:09.827 
```

Also, looking at the stacktrace, many goroutines seems leaked (stacktrace contains 889 `net/http.(*Transport).dialConn` routines).

--- a/integration-cli/docker_cli_swarm_test.go
+++ b/integration-cli/docker_cli_swarm_test.go
@@ -1576,7 +1576,7 @@ func (s *DockerSwarmSuite) TestSwarmServicePsMultipleServiceIDs(c *check.C) {
 func (s *DockerSwarmSuite) TestSwarmPublishDuplicatePorts(c *check.C) {
 	d := s.AddDaemon(c, true, true)
 
-	out, err := d.Cmd("service", "create", "--publish", "5000:80", "--publish", "5001:80", "--publish", "80", "--publish", "80", "busybox", "top")
+	out, err := d.Cmd("service", "create", "--publish", "5005:80", "--publish", "5006:80", "--publish", "80", "--publish", "80", "busybox", "top")
 	c.Assert(err, check.IsNil, check.Commentf(out))
 	id := strings.TrimSpace(out)
 
@@ -1588,8 +1588,8 @@ func (s *DockerSwarmSuite) TestSwarmPublishDuplicatePorts(c *check.C) {
 	out, err = d.Cmd("service", "inspect", "--format", "{{.Endpoint.Ports}} len={{len .Endpoint.Ports}}", id)
 	c.Assert(err, check.IsNil, check.Commentf(out))
 	c.Assert(out, checker.Contains, "len=4")
-	c.Assert(out, checker.Contains, "{ tcp 80 5000 ingress}")
-	c.Assert(out, checker.Contains, "{ tcp 80 5001 ingress}")
+	c.Assert(out, checker.Contains, "{ tcp 80 5005 ingress}")
+	c.Assert(out, checker.Contains, "{ tcp 80 5006 ingress}")
 }
 
 func (s *DockerSwarmSuite) TestSwarmJoinWithDrain(c *check.C) {

